index.js

```diff
@@ -104,7 +103,7 @@ exports.lookup_maxmind = function (next, connection) {
     agg_res.city = loc.city.names.en;
     if (plugin.cfg.show.city) show.push(loc.city.names.en);
   }
-  if (loc.location && loc.location.latitude) {
+  if (loc.location && !isNaN(loc.location.latitude)) {
```

Comment:
heh, so you're fixing here and in another PR?

Reply:
I'm still learning git and didn't realize that the isNaN version was submitted. I'd delete this PR but haven't figured out how to do so.